src/components/pages/user-profile.yml

```diff
@@ -1,4 +1,19 @@
 userProfile:
+  sendContactRequest:
+    en: Send contact request
+    de: Kontaktanfrage schicken
+  contactRequestModalInfoMarkdown:
+    en: User **{displayName}** will receive an email about your intention to get in contact. Provide an email address to which they can write you.
+    de: Benutzer **{displayName}** bekommt eine E-Mail mit deiner Absicht um Kontakt aufzunehmen. Gib eine E-Mail-Adresse, an die der Benutzer dir schreiben kann.
```

Comment:
Benutzer **{displayName}** bekommt eine E-Mail mit deiner Kontaktanfrage. Bitte gib eine E-Mail-Adresse an, unter der du gerne von **{displayName}** kontaktiert werden möchtest.

Reply:
Corrected, thank you 🙏